runit-nodejs/README.md

```diff
@@ -1,56 +0,0 @@
-socrata/runit
```

Comment:
This info should land somewhere. The runit containers are substantially different than the way socrata/base works.